scripts/directtools/__init__.py

```diff
@@ -626,7 +626,8 @@ def plotcuts(direction, workspaces, cuts, widths, quantity, unit, style='l', kee
                     markerStyle, markerIndex = _choosemarker(markers, markerIndex)
                 realCutCentre, realCutWidth = _cutcentreandwidth(line)
                 label = _label(ws, realCutCentre, realCutWidth, len(workspaces) == 1, len(cuts) == 1, len(widths) == 1, quantity, unit)
```

Comment:
also while you are in the file this line is too long as well

Reply:
Got it ! It should be all good now